LEAF_Request_Portal/api/controllers/SystemController.php

```diff
@@ -152,8 +152,8 @@ public function delete($act)
         $this->index['DELETE']->register('system', function ($args) {
         });
 
-        $this->index['DELETE']->register('system/files/[text]', function ($args) use ($db, $login, $system) {
```

Comment:
Would there be any custom code that this change would break?

Reply:
I don't think so.  It looked like that endpoint was used only by the file manager for files stored there.